Makefile

```diff
@@ -23,6 +33,20 @@ clean-mocks:
 	rm -f test/mocks/*.go forward/mock_*.go
 	rm -rf test/thrift/pingpong/
 
+lint:
+	@:>/tmp/lint.log
+
+	@-golint ./... | grep -Ev '(test|gen-go)/' | tee -a /tmp/lint.log
+
+	@for pkg in $(PKGS); do \
+		{ \
+			 find $$pkg -maxdepth 1 -mindepth 1 -type f -name '*.go' \
+				-exec go tool vet -printfuncs Logf:3 {} + 2>&1 ; \
+		} | tee -a /tmp/lint.log ; \
+	done;
+
+	@[ ! -s /tmp/lint.log ]
+
```

Comment:
The thing I don't like about this script is that it takes a package (or packages) as an argument, yet runs `golint` across `./...`.

There are several solutions, but for the sake of not sinking more time into this I'm just gonna go with what we've got.

I've updated the PR to output to `lint.log` and will clean it up afterwards.

We can mark this as something to be improved in the future but I think for now we should just move on.

Thank you for suggestions though; you're quite right about the /tmp thing.